packages/flutter/lib/src/gestures/drag.dart

```diff
@@ -63,10 +63,22 @@ bool _isFlingGesture(Velocity velocity) {
 }
 
 abstract class _DragGestureRecognizer<T extends dynamic> extends OneSequenceGestureRecognizer {
+  /// A pointer has contacted the screen and might begin to move.
   GestureDragDownCallback onDown;
+
+  /// A pointer has contacted the screen and has begun to move.
   GestureDragStartCallback onStart;
+
+  /// A pointer that is in contact with the screen and moving has moved again.
   _GesturePolymorphicUpdateCallback<T> onUpdate;
+
+  /// A pointer that was previously in contact with the screen and moving is no
+  /// longer in contact with the screen and was moving at a specific velocity
+  /// when it stopped contacting the screen.
   GestureDragEndCallback onEnd;
+
+  /// Signature for when the pointer that previously triggered [onDown] did not
```

Comment:
"Signature" sounds like the description of a typedef rather than the description of a property; was this a cut and paste mistake?